packages/model/src/team.ts

```diff
@@ -30,6 +31,7 @@ export interface TeamMember {
   email: string;
   role: TeamRole;
   avatarUrl?: string;
+  labs?: Lab[];
```

Comment:
Should this be optional? Could it be [] if undefined in the transformer?

Reply:
`TeamMembers` is created from `RestUser` or `GraphqlUser`. The first one doesn't have a labs field.

https://github.com/yldio/asap-hub/blob/74abb0af41ae1979d653ddc5e24a7f0218753853/apps/asap-server/src/controllers/teams.ts#L138
https://github.com/yldio/asap-hub/blob/74abb0af41ae1979d653ddc5e24a7f0218753853/apps/asap-server/src/controllers/teams.ts#L105